### PR TITLE
Add ability to control timeout for PHPUnit process

### PR DESCRIPTION
**Pull request by sibprogrammer**

### CODE REVIEW

File: src/Screens/Phpunit.php [hunk 1]
```diff
@@ -18,13 +18,16 @@ class Phpunit extends Screen
...
+    private $phpunitTimeout;
+
     public function __construct(array $options)
     {
         $this->options = $options;
 
         $this->phpunitArguments = $options['phpunit']['arguments'] ?? '';
-
         $this->phpunitBinaryPath = $options['phpunit']['binaryPath'] ?? self::DEFAULT_BINARY_PATH;
+        $this->phpunitTimeout = $options['phpunit']['timeout'] ?? 60;
```
Comment (freekmurze) [inline src/Screens/Phpunit.php]: Is `60` the default being used if `setTimeout` is not called on `Process`?

Comment (sibprogrammer) [reply src/Screens/Phpunit.php]: Yes, it is.

**Comment by sibprogrammer** [comment]
FYI styleci warnings have been fixed

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**